realm/src/main/java/io/realm/internal/SharedGroup.java

```diff
@@ -234,15 +234,16 @@ public void reserve(long bytes) {
 
     private native void nativeRollback(long nativePtr);
 
-    private native long createNative(String databaseFile,
+    private native long nativeCreate(String databaseFile,
```

Comment:
renaming a jni method should be reflected in the header and cpp file as well - can't see that's updated?